src/properties.c

```diff
@@ -165,7 +172,7 @@ static int set_modbus_source_properties(struct knot_thing *thing,
 
 	*bit_offset = bit_offset_aux;
 	*reg_addr = reg_addr_aux;
-
```

Comment:
Please, do not remove this line in the commit 158cdf10157ed1e0a6d950576bc56ce5f77c8550.